react-demo-app/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "react-demo-app",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "@reduxjs/toolkit": "^2.2.5",
+    "@vitejs/plugin-basic-ssl": "^1.1.0",
+    "genesys-cloud-webrtc-sdk": "file:..",
+    "genesys-spark-components": "^4.63.2",
+    "genesys-spark-components-react": "^4.63.2",
```

Comment:
I tried very very very very hard to not use spark, but for branding purposes (icons/typography) I brought it in.